fdm-app/app/components/custom/fertilizer-applications/form.tsx

```
@@ -0,0 +1,182 @@
+import { FormSchema } from "./formschema"
+import { Combobox } from "../combobox"
+import { Input } from "@/components/ui/input"
+import { cn } from "@/lib/utils"
+import { format } from "date-fns"
+import type { FertilizerOption } from "./types.d"
+
+export function FertilizerApplicationForm({
+    options,
+    action,
+    fetcher,
+}: { options: FertilizerOption[]; action: string; fetcher: any }) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)